redisinsight/ui/src/constants/redisVersions.ts

```diff
@@ -0,0 +1,5 @@
+export const RedisVersions = {
```

Comment:
We have a file with constants for different version. Please move there
`commandsVersions.ts`